### ISSUE TITLE
Suppress profile summary in output for resource packs

### ISSUE THREAD

**Issue by clintoncwolfe** [inspec/inspec#3203]
### 🎛 Description

**🙋 feature request

(on behalf of @trevorghess)

When a [resource pack](https://www.inspec.io/docs/reference/glossary/#resource-pack) is used to pull in custom resources and no tests, the test output nevertheless reports on the test count (zero) for the resource pack.   That leads to confusion.

As we increasingly encourage the use of resource packs for distributing custom resources, this becomes more important.

### 🌍 InSpec and Platform Version

2.2.x

### 🤔 Replication Case

Running any profile that is dependent on a resource pack will cause the spurious output.

```
C:\projects\inspec\inspec [master = +0 ~2 -0 !]> bundle exec inspec exec C:\projects\stuartpreston\inspec_azure_testing\v3\profile\ -t azure://....
Profile: inspec-azure-test (resource pack) (inspec-azure-test)
Version: 0.1.0
Target: azure://....
Resource Groups with name == "test-rg-01"
 [PASS] should exist
Profile: Azure Resource Pack (inspec-azure)
Version: 1.0.0
Target: azure://....
 No tests executed.
Test Summary: 1 successful, 0 failures, 0 skipped
```

### 💁 Possible Solutions

I suggest we omit CLI-mode output from profiles with no controls if at least one other profile has at least one control.

**Comment by aaronlippold** [comment]
We have also seen this when I've pulled in a resource pack and utilize the resources but the resource pack had control tests that I didn't use. Linux patches is a good example of this where if you only use the resources but not the controls from the pack those controls report out no data but end up in the Jason report which leads to confusion.

**Comment by clintoncwolfe** [comment]
This was addressed in #5373
